Docs/source/index.rst

```diff
@@ -3,19 +3,23 @@
 WarpX
 -----
 
-WarpX is an advanced **electromagnetic & electrostatic Particle-In-Cell** code.
+WarpX is an advanced **Particle-In-Cell** code.
 
 It supports many features including:
 
-    - Perfectly-Matched Layers (PML)
-    - Boosted-frame simulations
-    - Mesh refinement
+    - Multiple types of field solvers (incl. electromagnetic, electrostatic)
+    - Various grid geometries (1D/2D/3D Cartesian, cylindrical, spherical)
+    - Multi-physics packages (incl. ionization, atomic and collisional physics, quantum electrodynamics)
```

Comment:
I repost and elaborate on my Slack comment here on
```
(incl. ionization, atomic and collisional physics, quantum electrodynamics)
```
pointing out that, in my opinion, whether or not the word "fusion" appears in the landing page of the WarpX docs may (or should?) be a bit of a strategic decision.

Reply:
I think it makes sense to mention `fusion` if we specifically call out `atomic`. 

It might make more sense to link to the highlights page in another paragraph, after the one saying WarpX is highly parallel. Even just adding
```
WarpX has been applied to a wide variety of science projects, see [highlights](https://warpx.readthedocs.io/en/latest/highlights.html).
```